nvflare/apis/client.py

```diff
@@ -17,6 +17,12 @@
 
 class Client:
     def __init__(self, name, token) -> None:
+        """Init Client.
```

Comment:
We might want to add some explanation to this class.

We can remove # super().__init__() since it is commented out.